assets/components/directDebit/directDebitPopUpForm/directDebitPopUpForm.scss

```diff
@@ -43,7 +49,10 @@
   font-weight: bolder;
   font-size: 24px;
   line-height: 30px;
-  max-width: 280px;
+  max-width: 260px;
+  @include mq($from: mobileMedium){
```

Comment:
Should this be `$from: mobileMedium $to: mobileLandscape`?

Reply:
Is that necessary? One just takes over from the other doesn't it?

Reply:
I think it is fine the way it is